cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ZookeeperConfiguration.java

```diff
@@ -34,6 +34,8 @@
         config.put("tickTime", "2000");
         config.put("initLimit", "5");
         config.put("syncLimit", "2");
+        config.put("standaloneEnabled", "false");
+        config.put("reconfigEnabled", "true");
```

Comment:
Havig this here suggests this can be override and is just default. I think this should be in forbidden options + ardcoded in `zookeeper_config_generator.sh`. That probably applies also to `4lw.commands.whitelist`, but I'm fine to leave that for other day.

Reply:
Agreed.